mesonbuild/wrap/wrap.py

```diff
@@ -625,6 +625,8 @@ def _get_git(self, packagename: str) -> None:
         if self.wrap.values.get('clone-recursive', '').lower() == 'true':
             verbose_git(['submodule', 'update', '--init', '--checkout', '--recursive', *depth_option],
                         self.dirname, check=True)
+        else:
```

Comment:
This is only a very very small improvement.

When testing with the decimal wrap (#13821), I changed the value of clone-recursive to 1. I run `--wipe` and meson errors out as intended (ERROR: clone-recursive: invalid value (use 'true')). I run `--wipe` again, and I don't get the error. Instead, meson tries to build the cmake project but errors because the submodules haven't been pulled.

Reply:
I understand why this is failing the tests. If `clone-recursive` is omitted from the wrap, the exception is getting raised. I'll try to fix that.

Reply:
I think it's questionable to call it an improvement. Some people may believe explicit is better than implicit, and be defining this with a value other than "true"...

Reply:
@eli-schwartz Oh.. I get what you mean I think. It took me a few minutes. I'll fix that so 'false' is also acceptable.